src/libraries/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -959,6 +960,40 @@ public void TestMessages()
             }).Dispose();
         }
 
+        // Tests that version event from DiagnosticSourceEventSource is fired.
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void TestVersion()
+        {
+            RemoteExecutor.Invoke(() =>
+            {
+                using (var eventSourceListener = new TestDiagnosticSourceEventListener())
+                {
```

Comment:
>          [](http://example.com/codeflow?start=2&length=9)

Should you have 
```C#
using (var diagnosticSourceListener = new DiagnosticListener("TestMessagesSource"))
```

as you are listening to `TestMessagesSource`?

Reply:
`TestMessagesSource` was there from some copy/paste. I switched it to just `""` to make it more clear.